lib/datadog/core/configuration/components.rb

```diff
@@ -126,6 +129,11 @@ def startup!(settings)
             end
           end
 
+          if settings.remote.enabled && old_state&.[](:remote)
+            # remote should be defined here
+            remote.start
+          end
```

Comment:
Why is this change needed?

Reply:
I use it for testing to avoid having to hit the app before it instruments code but it does not belong in this PR, I will revert it.